docs/analyse.md

```diff
@@ -40,10 +40,307 @@ watch --interval 30 python -m neps.status ROOT_DIRECTORY
 
 ## Visualizations
 
+### 1. CLI commands
+
 To generate plots to the root directory, run
 
 ```bash
 python -m neps.plot ROOT_DIRECTORY
 ```
 
 Currently, this creates one plot that shows the best error value across the number of evaluations.
+
+### 2. TensorBoard
+
+#### Introduction
+
+[TensorBoard](https://www.tensorflow.org/tensorboard) serves as a valuable tool for visualizing machine learning experiments, offering the ability to observe losses and metrics throughout the model training process. In NePS, we use this powerful tool to show metrics of configurations during training in addition to comparisons to different hyperparameters used in the search for better diagnosis of the model.
+
+#### The Logging Function
+
+The `tblogger.log` function is invoked within the model's training loop to facilitate logging of key metrics.
+
+- **Signature:**
+```python
+tblogger.log(
+    loss: float,
+    current_epoch: int,
+    write_summary_incumbent: bool = False,
+    write_config_scalar: bool = False,
+    write_config_hparam: bool = True,
+    extra_data: dict | None = None
+)
+```
```

Comment:
is it possible to do a ```python``` block here and show that this function can only be used when inside a `run_pipeline()`.
Of course, a dummy run_pipeline with `...` before and after the `tblogger.log` call.